### DISCUSSION TITLE
Support upload_to_testflight for macOS TestFlight

### DISCUSSION THREAD

**Discussion by Igralino** [fastlane/fastlane#19298] [Ideas] [answered]
A few days ago Apple released TestFlight for macOS ([news on 9to5mac](https://9to5mac.com/2021/08/24/apple-now-letting-developers-submit-mac-beta-apps-with-xcode-13-ahead-of-testflight-for-macos/)) and I think that many developers will be glad to see possibility to upload and manage macOS builds for macOS TestFlight via fastlane. 

Do you have any plans to support macOS TestFlight?

**Comment by lucgrabowski** [comment]
Hi, yes, there is even a pull request opened with implementation allowing for uploading mac apps to TestFlight: https://github.com/fastlane/fastlane/pull/19296.

**Reply by MikeChongCan** [reply]
Hi, when will the next version be released?

**Comment by lucgrabowski** [comment, marked as answer]
It's supported in the newest release. 🎉

**Reply by MikeChongCan** [reply]
Maybe we need a new release for this one? https://github.com/fastlane/fastlane/pull/19319

**Reply by lucgrabowski** [reply]
It's just been released in 2.193.1

**Reply by Igralino** [reply]
@lucgrabowski I wrote the other issue connected with uploading testflight builds to AppStoreConnect: https://github.com/fastlane/fastlane/pull/19319#issuecomment-911763341

Could you look at it please? :)
